app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
Use i18n here

Reply:
done

Reply:
The comment is not fix yet
please refs: https://guides.rubyonrails.org/i18n.html#setup-the-rails-application-for-internationalization to apply it